### DISCUSSION TITLE
Add Net Remote to SDRpp

### DISCUSSION THREAD

**Discussion by df6dbf** [AlexandreRouma/SDRPlusPlus#243] [Ideas]
In the older SDR+ Releases you can add a Net Remote Plugin. IMHO its a good idea to add net remote to SDRpp. I use it for Weather Ballon Monitoring with RS41-Tracker (drives SDRsharp via Network).

73s Holger

**Comment by AlexandreRouma** [comment]
SDR++ already has rigctl integration which can be used for control via network.

**Reply by df6dbf** [reply]
Thanks for the quick reply, found it finally but it seems it doesnt work as the old fashioned "NetRemote"-Plugin. Got no Frequency Exchange between RS41-Tracker and SDRpp (Port Number is equal).

Will ask Diego, IW1GIS for change to the more actual Rigcrl.

73s Holger
